### ISSUE TITLE
Share sheet

### ISSUE THREAD

**Issue by 2br-2b** [howeyc/crocgui#17]
Hello!

Thank you so much for this app! It makes sharing files from my computer and phone so much easier!

My only problem is just a minor inconvenience. From other apps, when I press Share, Croc doesn't show up. It would be really nice if Croc showed up there with my other messaging apps!

Regardless, great job with this app! It's much quicker for me than having to open Signal Desktop on my laptop every time I need to share a file,

**Comment by howeyc** [comment]
Yeah, this is an interesting idea.

**Comment by arminus** [comment]
That would be very convenient!

**Comment by mrsalo** [comment]
This would greatly enhance the usability, +1

**Comment by drmdshahid** [comment]
Much needed.

**Comment by zanona** [comment]
Thanks so much for Croc for Android! It's a really helpful tool.
I support the share sheet request and also the ability to share a received file rather than only allowing it to download to the device. The benefits are that we will probably be already be able to save it somewhere from the share sheet (Google Drive, etc), as well as forwarding to any messaging application, for example. So it reduces the process of saving to the device and then attaching the file on the app intended. 

Thanks again :+1:

**Comment by howeyc** [comment]
Yes, I definitely understand how useful the feature would be, to be able to send to/from croc and other apps within Android. I just don't know how I would do it. Yes, I am aware of the Android Intents system, but the MainActivity is in the Go Fyne library I'm using and I'm unsure how I would be able to access the layers within Android to do the sharing.

**Comment by zanona** [comment]
Yes, I've tried to find out on Fyne slack channel but it seems that it's currently not easily achievable unless some custom bindings are added. Hopefuly it will be implemented at some point in the future in the form of a seamless integration for Android and iOS share sheet. At the moment, we would be a bit stuck as you've mentioned, unfortunately.

Thanks for following uo, though.

**Comment by abakum** [comment]
Try "future" https://github.com/abakum/crocgui/commit/a9ea70d3aa4e25f944e40c407e9a9e17e55793b9
